packages/ncids-react/package.json

```diff
@@ -48,6 +49,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "rimraf": "^3.0.2",
```

Comment:
shared dependency?

Reply:
Just for posterity as I noted in Slack, if the package uses the thing, it should be in the dependencies. This is even if a dependent package also depends on it. This way if something changes elsewhere, this package will not break. Yarn should handle managing one version of rimraf for all of the packages that need it. (So it does become shared.)

Reply:
I will also note that Paul pointed out it is not in ncids-css, so I will go add it there.